wolfcrypt/src/evp.c

```diff
@@ -7085,9 +7587,18 @@ int wolfSSL_EVP_MD_type(const WOLFSSL_EVP_MD* type)
         WOLFSSL_ENTER("wolfSSL_EVP_Cipher");
 
         if (ctx == NULL || ((src == NULL || dst == NULL) &&
-            (ctx->cipherType != AES_128_GCM_TYPE &&
+            (TRUE 
+            #ifdef HAVE_AESGCM
```

Comment:
Please reduce indent on the `#if` preprocessors by 4 spaces. Makes easier to read.